addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3338,7 +3338,231 @@ msgctxt "#802"
 msgid "%s of %s available"
 msgstr ""
 
-#empty strings from id 803 to 849
+#. Label of spinner control for choosing the timer type (one time, repeating, ...)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#803"
+msgid "Timer type:"
```

Comment:
146 has a trailing colon. I just removed all the colons from the new timer settings dialog labels.